Sources_v3/StreamChatUI/ChatChannelList/ChatChannelAvatarView.swift

```diff
@@ -0,0 +1,59 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+
+open class ChatChannelAvatarView: UIView {
+    
+    // MARK: - Subviews
+    
+    public let imageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        return imageView
+    }()
+    
+    // MARK: - Init
+    
+    override public init(frame: CGRect) {
+        super.init(frame: frame)
+        commonInit()
+    }
+    
+    public required init?(coder: NSCoder) {
+        super.init(coder: coder)
+        commonInit()
+    }
+    
+    private func commonInit() {
+        setupLayout()
+        setupAppearance()
+    }
+    
+    // MARK: - Layout
+    
+    override open func layoutSubviews() {
+        super.layoutSubviews()
+        
+        layer.cornerRadius = bounds.width / 2
+    }
+    
+    // MARK: - Public
+    
+    open func setupAppearance() {
+        layer.masksToBounds = true
+    }
+    
+    open func setupLayout() {
+        addSubview(imageView)
+        NSLayoutConstraint.activate([
+            imageView.leadingAnchor.constraint(equalTo: leadingAnchor),
+            imageView.trailingAnchor.constraint(equalTo: trailingAnchor),
+            imageView.topAnchor.constraint(equalTo: topAnchor),
+            imageView.bottomAnchor.constraint(equalTo: bottomAnchor),
+            
+            widthAnchor.constraint(equalTo: heightAnchor, multiplier: 1)
+        ])
```

Comment:
yeah, we really need some helper for those

Reply:
added in 0ef92e65959e905564a5a1660f6fd50fc7ca2e15, used in d53402bb0a05484f8f7b2dce2e1c3b55278815a2